app/services/github_scrape_service.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class GithubScrapeService
```

Comment:
@Br3nda these red lines saying, tests don't exist for them, I've tried writing some. Can you have a look at it when you have time thanks!